src/io/openshift/plugins/analytics.groovy

```diff
@@ -1,10 +1,29 @@
 package io.openshift.plugins
 
 import io.openshift.Events
+import io.openshift.Utils
 
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
+        def image = config.runtime() ?: ''
```

Comment:
```groovy

def runtime = config.runtime() 
if (!runtime) {
   echo "WARNING: skipping analytics since runtime isn't set"
   return
}
```
rt = config.runtime() ?: '' `?: ''` isn't needed.